config/constants.js

```diff
@@ -20,6 +21,9 @@ exports.PASSWORD_SALT = 10;
 exports.BAD_CREDENTIALS = 'User Credentials are not valid';
 exports.SIGN_IN_SUCCESSFUL = 'Sing in was successful';
 exports.INVALID_TOKEN = 'Unauthorized for this request';
+exports.ADMIN_ROUTE_PATTERN = /(^|[/])(admin)([?/]|$)+/;
```

Comment:
maybe there will be routes that only admin can consume, but not necessary includes the string "admin" so I think this approach can become poorly maintainable. We can discuss.

Reply:
Ok, as I mentioned before, I switched to a middleware based strategy.